src/main/java/org/nebula/contrib/ngbatis/NgbatisContextInitializer.java

```diff
@@ -58,7 +58,9 @@ private ParseCfgProps readParseCfgProps(ConfigurableEnvironment environment) {
       .setStatementStart(environment.getProperty("cql.parser.statement-start"))
       .setResourceRefresh(
         environment.getProperty("cql.parser.statement-start", Boolean.class)
-      );
+      )
+      .setSessionLifeLength(environment.getProperty("cql.parser.session-life-length", Long.class))
+      .setCheckFixedRate(environment.getProperty("cql.parser.check-fixed-rate", Long.class));

```

Comment:
As above. `nebula.ngbatis.session-life-length` ?

Reply:
The ngbatis field is added in NebulaJdbcProperties to customize the ngbatis parameter.